schema.sql

```diff
@@ -61,3 +61,8 @@ CREATE TABLE visits (
     CONSTRAINT pk_visits PRIMARY KEY (animals_id, vets_id, date_of_visit)
 );
 
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+   create index animal_id_desc on visits(animal_id desc);
+   create index vet_id_desc on visits(vet_id desc);
+   create index email_desc on owners(email desc);
```

Comment:
[ OPTIONAL ]
This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project.


As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:


>CREATE INDEX ON owners ((lower(email)));

Reply:
Thank you @tugsem